pkg/validation/limits.go

```diff
@@ -91,6 +91,9 @@ type Limits struct {
 	CompactorPartialBlockDeletionDelay model.Duration `yaml:"compactor_partial_block_deletion_delay" json:"compactor_partial_block_deletion_delay"`
 	CompactorDownsamplerEnabled        bool           `yaml:"compactor_downsampler_enabled" json:"compactor_downsampler_enabled"`
 
+	// Tenant settings.
+	TenantSettingsOverrides map[string]string `yaml:"tenant_settings_overrides" json:"tenant_settings_overrides"`
+
```

Comment:
I guess we should also register this as a CLI flag for consistency with other limits. `RelabelRules` is a good example of how a structured configuration may be handled. Also, a list representation of settings will probably be easier to manage